src/main/java/frc/robot/RobotManager.java

```diff
@@ -24,6 +25,8 @@
  */
 public class RobotManager extends LoggedRobot {
 
+	public static int roborioCycles;
```

Comment:
Consider making this not static

Reply:
i need to address it from timeutils so it has to be

Reply:
this ruins the design of `RobotManager` not being a singleton

Reply:
i dont think its ruining the design, why do u think so?

Reply:
@YoniKiriaty

Reply:
I'm pretty sure that's the point of static

Reply:
Yea, but if it didn't get to the part that it gives it a value, then exceptions will be thrown since you would be trying to access an int without a value

Reply:
Usually, when we have static values they are final. It really is a problem if such thing happens in other classes. Usually there is no problem with that since the code is built in a way that it gets initialized before other classes get to use it but since this is robot template and the people of the future would like to use it, we cant make sure they will not change something in the code that would use the int before it initializes